### PR TITLE
Update Source.actorRef sample to not use deprecated method 

### PR DESCRIPTION
**Pull request by johanandren**
References #28679

### CODE REVIEW

**Review by raboof — APPROVED** [review]
Nice, good comments

File: akka-docs/src/test/scala/docs/stream/operators/SourceOperators.scala [hunk 1]
```diff
@@ -29,23 +30,31 @@ object SourceOperators {
...
+    val source: Source[Any, ActorRef] = Source.actorRef(
+      {
+        case Done =>
+          // complete stream immediately if we send it Done
+          CompletionStrategy.immediately
+      },
+      // never fail the stream because of a message
+      PartialFunction.empty,
+      bufferSize,
+      OverflowStrategy.dropHead)
```
Comment (ennru) [inline akka-docs/src/test/scala/docs/stream/operators/SourceOperators.scala]: Use named arguments?

Comment (johanandren) [reply akka-docs/src/test/scala/docs/stream/operators/SourceOperators.scala]: Done

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**